repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -91,7 +91,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
       inputStream.close()
       Some(defineClass(name, bytes, 0, bytes.length))
     } catch {
-      case e: Exception => None
+      case e: FileNotFoundException =>
+        // We did not find the class
+        logDebug(s"Did not load class $name from REPL class server at $uri", e)
```

Comment:
OK, it's rarely a remote server. I'd just log the URI.
Also this is borderline non-trivial enough to need a JIRA, although, I think this doesn't change any behavior, just logs things.